scapy/contrib/tcpros.py

```diff
@@ -0,0 +1,732 @@
+# This file is part of Scapy
+# Scapy is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# any later version.
```

Comment:
Almost all parts of Scapy are using GPLv2 only License.

Reply:
Simplified license at 40cf09373065b8f15db88535424b6c226fd62b1d